uaaextras/webapp.py

```diff
@@ -495,8 +495,8 @@ def signup():
                 return redirect(url_for("logout"))
             else:
                 logging.exception("An error occured communicating with UAA")
-        except Exception:
-            logging.exception("An error occured during the invite process")
+        except Exception as e:
```

Comment:
Upon further review, I think these changes are unnecessary because `logging.exception` prints a stack trace with the error already?

https://stackoverflow.com/a/5191885